OfficeMicroService/Application/Services/DTO/OfficeForChangeDTO.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson;
+using System.ComponentModel.DataAnnotations;
+
+namespace OfficeMicroService.Application.Services.DTO
```

Comment:
Good job! But let's move DTOs folder upper to the Application folder, not inside the Services folder